packages/firebase_crashlytics/firebase_crashlytics/example/lib/main.dart

```diff
@@ -61,14 +61,6 @@ class _MyAppState extends State<MyApp> {
           .setCrashlyticsCollectionEnabled(!kDebugMode);
     }
 
-    // Pass all uncaught errors to Crashlytics.
-    Function originalOnError = FlutterError.onError;
-    FlutterError.onError = (FlutterErrorDetails errorDetails) async {
-      await FirebaseCrashlytics.instance.recordFlutterError(errorDetails);
-      // Forward to original handler.
-      originalOnError(errorDetails);
-    };
```

Comment:
Doing so will cause `presentError` to be called twice. The preferred way of setting this is like it's done above https://github.com/FirebaseExtended/flutterfire/blob/352ea9c2a1496d5bd9127274be5c68d90a2ee953/packages/firebase_crashlytics/firebase_crashlytics/example/lib/main.dart#L27